### PR TITLE
Add "What's Your Sign?" 1.2.1

### PR DESCRIPTION
**Pull request by listochkin**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

App uses same installer technique as all other Objective-See apps: zip contains an '*_Installer.app' that has to be manually run to install a helper for Finder.

### CODE REVIEW

File: Casks/whatsyoursign.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'whatsyoursign' do
```
Comment (vitorgalvao) [inline Casks/whatsyoursign.rb]: Corrected this.

File: Casks/whatsyoursign.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'b05363ce2106e23eda95caef1f812fb6cb1dc8f983dee035a4730afe35fc13d6'
+
+  # bitbucket.org/objective-see was verified as official when first introduced to the cask
+  url "https://bitbucket.org/objective-see/deploy/downloads/WhatsYourSign_#{version}.zip"
+  name "What's YourSign?"
+  homepage 'https://objective-see.com/products/whatsyoursign.html'
+
+  installer manual: 'WhatsYourSign_Installer.app'
+
+  uninstall delete: '/Applications/WhatsYourSign.app'
```
Comment (vitorgalvao) [inline Casks/whatsyoursign.rb]: It’s unlikely moving an app is the only thing the installer does. Please search their website or contact the developer for a thorough uninstall procedure.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/whatsyoursign.rb (hunk 1 above)
Comment (listochkin) [inline Casks/whatsyoursign.rb]: `"$(brew --repository)/Library/Taps/caskroom/homebrew-cask/developer/bin/generate_cask_token" "What's Your Sign?"` gave me `whats-your-sign`

File: Casks/whatsyoursign.rb (hunk 2 above)
Comment (listochkin) [inline Casks/whatsyoursign.rb]: Yep, you're right. Turned out one has to run the `WhatsYourSign_Installer.app` again. How can I express in in Cask-file?

**Comment by Numbermaniac** [comment]
According to their website:

Install:
a) copies WhatsYourSign.app into the /Applications directory
b) registers the Finder extension WhatsYourSign.appex
(found within WhatsYourSign.app/Contents/Plugins) 

Uninstall:
a) unregisters the Finder extension, WhatsYourSign.appex
b) removes the /Applications/WhatsYourSign.app directory (which also removes WhatsYourSign.appex)

**Comment by vitorgalvao** [comment]
Closing for lack of reply.

**Pull request closed without merging**